lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Adding `ProjectReference` doesn't change behavior of test project. I suggest we keep tests independent of builds, so that we have same functionality as we did when using vstest.console.exe (actually improved because now we have single command to run all tests).